src/controller/include/controller/controller.h

```diff
@@ -80,6 +82,10 @@ class Controller : public ControllerInterface {
 	 * @see ControllerInterface#release_button
 	 */
 	void release_button(Button button) override;
+
+	uint64_t get_ticks() override;
+
+	std::shared_ptr<debugger::DebuggerCore> debugger_core;
```

Comment:
Why does this need to be here? This really breaks the design. Lets discuss this.